### PR TITLE
feat/Quantum art visualization

### PR DESCRIPTION
**Pull request by gcattan**
This is a PR, which explores a topic of quantum computing called quantum art, that is, some kind of artistic visualization of quantum.
We can find some examples [here](https://medium.com/qiskit/how-i-use-quantum-computing-to-create-bubble-art-d6c01f3ec2e), where noise inside quantum calculation is used to generate bubbles. The number of bubbles and the size of each bubble then directly depend on the noise model inside the quantum computer.

In this PR, the fitted weight of the variational circuit inside VQC is displayed in a spiral. The number of "branches" in the spiral depends on the number of parameters, and the "larger" of each branch depends on the variability of the weights.

Figure_1

### CODE REVIEW

File: pyriemann_qiskit/datasets/utils.py [hunk 1]
```diff
@@ -98,12 +98,19 @@ def get_mne_sample(n_trials=10, include_auditory=False):
     return X, y
 
 
-def get_qiskit_dataset():
+def get_qiskit_dataset(n_samples=30):
```
Comment (toncho11) [inline pyriemann_qiskit/datasets/utils.py:101]: Wouldn't be confusing for someone who wants the entire dataset to get only 30 samples when calling the function without parameters?

File: examples/toys_dataset/plot_quantum_art_vqc.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+
+print(__doc__)
+
+###############################################################################
+# In this example we will display weights variability of the parameter inside
+# the variational quantum circuit which is used by VQC.
+#
+# The idea is simple :
+# - We initialize a VQC with different number of parameters and number of samples
```
Comment (toncho11) [inline examples/toys_dataset/plot_quantum_art_vqc.py:28]: Add . at the end of this sentence.

File: examples/toys_dataset/plot_quantum_art_vqc.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+# - We train the VQC a couple of time and we store the fitted weights.
```
Comment (toncho11) [inline examples/toys_dataset/plot_quantum_art_vqc.py:29]: time -> times

File: examples/toys_dataset/plot_quantum_art_vqc.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+# - We compute variability of the weight and display it in a fashion way.
```
Comment (toncho11) [inline examples/toys_dataset/plot_quantum_art_vqc.py:30]: fashion -> fashionable

**Review by toncho11 — COMMENTED** [review]
Very good.

File: pyriemann_qiskit/datasets/utils.py (hunk 1 above)
Comment (gcattan) [reply pyriemann_qiskit/datasets/utils.py:101]: This dataset is generated. So you have to specify somehow the number of samples you want.

Comment (toncho11) [reply pyriemann_qiskit/datasets/utils.py:101]: I see. Then the name could be "generate_qiskit_dataset".

Comment (gcattan) [reply pyriemann_qiskit/datasets/utils.py:101]: Yes, good point.

**Pull request merged**